codeguru/views.py

```diff
@@ -54,6 +51,16 @@ def group(request, id=None):
 
                 return redirect("group", id=new_group.id)
 
+            if CgGroup.objects.all().filter(name=form.data["name"]).exists():
+                return error(request, gettext("Group already exists. try choosing another name."))
+            else:
+                return error(
+                    request,
+                    gettext(
+                        "Group name is invalid. Groups names may only include alphanumerical characters and underscores"
+                    ),
+                )

```

Comment:
In theory, we can also use `Form.errors` to get the exact error message the form raised. Not critical if it's a lot of work or complicates things.

Also, what if the submitted form has a different error? (for example - no center)

Reply:
tried using `form.errors`  but I could not figure out how to get it to translate "name"
![image](https://github.com/user-attachments/assets/8e7ffdd2-5fe5-4776-a1d9-5630b9402225)
added case for invalid name, if the error is not invalid/taken name, it returns `form.errors` (which probably won't be fully translated)